### ISSUE TITLE
Winmm error with latest version of System.Speech package

### ISSUE THREAD

**Issue by mystborn** [dotnet/runtime#69776]
### Description

When using the System.Speech v7.0.0-preview.4.22229.4 package, trying to call any method on a SpeechSynthesizer throws the following exception:

```
System.EntryPointNotFoundException: 'Unable to find an entry point named 'waveOutGetDevCaps' in DLL 'winmm.dll'.
```

### Reproduction Steps

using System.Speech.Synthesis;

using (var speech = new SpeechSynthesizer())
{
    // Throws exception right away
    speech.SetOutputToDefaultAudioDevice();
    speech.SelectVoiceByHints(VoiceGender.Female);
    speech.SpeakProgress += (s, e) =>
    {
        Console.Write($"{e.Text} ");
    };

    speech.Speak("This is a test string. Thanks for listening.");

    Console.WriteLine("Finished speaking...");
    Console.ReadLine();
}

### Expected behavior

No exception is thrown and methods are able to run as expected.

### Actual behavior

EntryPointNotFoundException is thrown

### Regression?

The package does work fine when using as late as 6.0.2-mauipre.22102.15

Only when upgrading to 7 does the exception start getting thrown.

### Known Workarounds

Downgrade the package to 6.x.x

### Configuration

.NET 6
Windows 10 (Build 19043)
All Architectures

**Comment by huoyaoyuan** [comment]
Should already be fixed by #68418

**Comment by danmoseley** [comment]
Indeed. My bad for not seeing this comment earlier and closing it.
